post-processing-code/general-functions/general_post_processing.py

```diff
@@ -0,0 +1,261 @@
+import numpy as np
+import pandas as pd
+
+
+# todo: have the post processing be unique functions to call, or use a class with attributes
+
+def measure_summary_float(data, descriptor_column_locs, draws_loc):
+    values = data[:, draws_loc].astype(float)
+    return np.column_stack(
+        (
+            data[:, descriptor_column_locs],
+            np.mean(values, axis=1),
+            np.percentile(values, 2.5, axis=1),
+            np.percentile(values, 5, axis=1),
+            np.percentile(values, 10, axis=1),
+            np.percentile(values, 25, axis=1),
+            np.percentile(values, 50, axis=1),
+            np.percentile(values, 75, axis=1),
+            np.percentile(values, 90, axis=1),
+            np.percentile(values, 95, axis=1),
+            np.percentile(values, 97.5, axis=1),
+            np.std(values, axis=1),
+            np.median(values, axis=1),
+        )
+    )
+
+def processSingleFile(
+    df: pd.DataFrame,
+    scenario: str,
+    iuName: str,
+    year_column_name = "year_id",
+    measure_column_name = "measure",
+    age_start_column_name = "age_start",
+    age_end_column_name = "age_end",
+    draw_names: str = [f'draw_{i}' for i in range(0, 200)],
+    prevalence_marker_name: str = "prevalence",
```

Comment:
💚 I like how these are provided as inputs - avoids hiding magic strings in the code.